app/Http/Controllers/Admin/DiagramController.php

```diff
@@ -94,27 +104,49 @@ public function generateDiagram(Request $request, $idWorkspace)
         }
         $locations = $workspace->locations;
         $colorLocation = [];
+
         foreach ($locations as $key => $location) {
             foreach ($location->seats as $id => $seat) {
+                $user_id = $this->userRepository->get();
+                $list_user_id = unserialize($seat->user_id);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 111</br>Variable "list_user_id" is not in valid camel caps format